inc/features/blocks/personalization/components/views.js

```diff
@@ -31,6 +31,8 @@ const Views = ( {
 		);
 	}
 
+	const conversionPercent = ( ( conversions / ( conversionsDenominator || total ) ) * 100 ).toFixed( 1 );
```

Comment:
`toFixed` could go into the formatting code if you want to separate calculation from display.